src/components/InLineNotification.tsx

```diff
@@ -127,6 +128,10 @@ const severityColors: Record<Severity, CustomColors> = {
     primary: Colors.errorDark,
     secondary: Colors.errorLight,
   },
+  [Severity.SoftInformational]: {
+    primary: Colors.black,
+    secondary: Colors.gray1,
+  },
```

Comment:
Is this going to be part of the design system and used in multiple places? If so, should we have design update the style guide? I see this component [here](https://www.figma.com/file/erFfzHvSTm5g1sjK6jWyEH/Working-Design-System-Components?node-id=493%3A11&mode=dev) and [here](https://www.figma.com/file/l6cnlCx2Ht2YSgnbBdzZph/App-Style-Guide?node-id=0%3A1&mode=dev). Not sure which one is the style guide we should be using

Reply:
I'll clarify, thanks i was unaware of those style guides